.github/workflows/platform-install-testing.yml

```diff
@@ -0,0 +1,48 @@
+name: Platform Testing Installation
+on:
+  push:
+    branches: [ master ]
+  repository_dispatch:
+    types: [qa]
+  workflow_dispatch:
```

Comment:
Isnt this supposed to checkout the segregated path ?

Reply:
All the platform UI tests do not run on PRs according to their set up, they only run on a push done on the module. So there it has no value to add segregated path to them.